algebraic-types/README.md

```diff
@@ -0,0 +1,7 @@
+
+Extra tools and samples for the Bril algebraic types extension.
+
+ - `briltc`: Bril type-checker with support for algebraic types
+ - `rs2bril`: compiler from Rust syntax to Bril
```

Comment:
Seems like this deserves to be at the top level too. It's a sibling to `ts2bril`.